base/range.jl

```diff
@@ -861,13 +861,15 @@ end
 median(r::Range{<:Real}) = mean(r)
 
 function in(x, r::Range)
+    applicable(-, x, first(r)) || return false
     n = step(r) == 0 ? 1 : round(Integer,(x-first(r))/step(r))+1
```

Comment:
The (preexisting) branch in the `n = ...` line, rather than the new `applicable(...` line :). To expand, down one path `n = 1`, whereas down the other `n = round(Integer, ...`. If the `round` yields some non-`Int` `Integer`, `n`'s type would depend on the branch? Best!